File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n=nums.size();
        
        int ans = (n*(n+1))/2;
        
        for(int i=0;i<nums.size();i++){
            ans=ans-nums[i];
        }
        return ans;
    }
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        
        int n=nums.length;
        
        int total = (n*(n+1))/2;
        
        for(int i=0;i<nums.length;i++){
            total -= nums[i];
        }
        
        return total;
    }
}